src/RegexHelper.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\ObjectEncryptor;
+
+use Keboola\ObjectEncryptor\Exception\ApplicationException;
+
+class RegexHelper
+{
+    public static function matchesVariable(string $value): bool
+    {
+        $regex = '/^{{\s?[a-zA-Z][a-zA-Z0-9_\-]*\s?}}$/';
```

Comment:
@MiroCillik nepodporujeme sice plnou syntaxi Mustache, ale tohle by se melo sjednotit aspon s https://github.com/keboola/platform-libraries/blob/2412c63dffcdde4dd0b3c656f5c14ee13b7a1c2f/libs/configuration-variables-resolver/src/SharedCodeResolver.php#L84 (na zacatku/konci muze byt vic mezer, opravdu musi klic zacinat pismenem?)

Plus teda nevim uplne, jak s tou `{{{` variantou. Shared code ji taky nepodporuje, ale v testech resovleru ji mame a Mustache ji zpracuje https://github.com/keboola/platform-libraries/blob/164d9ec3a7c1545dc84beb8e11dea5a57c8c31d2/libs/configuration-variables-resolver/tests/VariableResolverTest.php#L446

Reply:
@pepamartinec good point, ze by to malo byt stejny. Jeste na to mrknu, to ze klic musi zacinat pismenem si zaved ty ve Vaultu :) ([viz zadani SOX-70](https://keboola.atlassian.net/browse/SOX-70)). 
I kdyz tohle neni nutne jenom pro Vault, stejne bych se toho drzel. 

Ty mezery opravim a ty 3ojity `{{{` to nevim, na to se musim podivat...

Reply:
> I kdyz tohle neni nutne jenom pro Vault, stejne bych se toho drzel.

no to ne, tady ten check musi pokryvat vsechno, co muze jit do Vaultu, shared code i konfig variables (takze tu nejoecnejsi moznou vec)